ansible/roles/kill-network/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
   shell: docker volume ls -qf dangling=true | xargs -r docker volume rm
 
 - name: remove all docker images
-  shell: docker images --no-trunc | awk '{ print $3 }' | xargs -r docker rmi
+  shell: docker image ls -q | awk '{ print $3 }' | xargs -r docker image rm
```

Comment:
What is the point of changing `images` to `image` and `rmi` to `image rm`?

The only thing which actually changes here is the `-q` argument instead of vs `--no-trunc`, which changes the output. Also, this won't work with the `awk` command anymore, because this will just show blanks as there is no 3rd field. You can remove the `awk` pipe entirely I think.